cmd/fleet/calendar_cron.go

```diff
@@ -184,14 +190,26 @@ func processCalendarFailingHosts(
 	hosts []fleet.HostPolicyMembershipData,
 	logger kitlog.Logger,
 ) error {
+	hosts = filterHostsWithSameEmail(hosts)
```

Comment:
Is it possible we filter a host with an existing calendar event? Or is the order of hosts and the filtering deterministic?

Reply:
The idea is to always pick the failing host with lower host ID.